pkg/remotewrite/prometheus_test.go

```diff
@@ -37,7 +37,7 @@ func TestMapSeries(t *testing.T) {
 }
 
 // buildTimeSeries creates a TimSeries with the given name, value and timestamp
-func buildTimeSeries(name string, value float64, timestamp time.Time) *prompb.TimeSeries {
+func buildTimeSeries(name string, value float64, timestamp time.Time) *prompb.TimeSeries { //nolint:unparam
```

Comment:
What is the error here?

Reply:
The linter says the `value` parameter has always the same value and can be removed, as the codebase has some todos left, I didn't want to assume anything about the further usage of this function and deactivated the lining locally.